shiny/global-sensitivity/load_ensemble.R

```diff
@@ -0,0 +1,43 @@
+#' @param workflow_dir PEcAn workflow directory
+#' @param settings PEcAn settings list
+#' @param variable Variable names to read, as a character vector
+#' @param quiet If TRUE, don't show status messages from `read.ensemble.output`
+load_ensemble <- function(workflow_dir, settings, variable, quiet=TRUE){
+    library(PEcAn.all)
+    library(ncdf4)
+    
+    # Load the model output
+    ## ANS -- NOTE: There may be a faster/better way to do this using built-in PEcAn functions
+    ## ANS -- or...should these be automatically stored somewhere?
+    
+    if(quiet) sink("/dev/null")
+    ensemble.output.raw <- read.ensemble.output(ensemble.size = NULL,
+                                                pecandir = workflow_dir,
+                                                outdir = settings$modeloutdir,
+                                                start.year = as.numeric(settings$ensemble$start.year),
+                                                end.year = as.numeric(settings$ensemble$end.year),
+                                                variable = variable)
+    if(quiet) sink()
+    ensemble.output <- data.frame(do.call(rbind, ensemble.output.raw))
+    
+    ## NOTE: read.ensemble.output only returns the mean value at each timestep.
+    ## If we want other statistics, they need to be either hard-coded (loop with read.output),
+    ## or read.ensemble.output needs to be modified.
```

Comment:
Good to know. Also, I think we should consider running `read.ensemble.output` on all variables after each ensemble run by default in the PEcAn workflow. Since it averages over time, I can't imagine the resulting data frames being all that big (n_vars x n_ensemble), and it would make this app work way faster.